### DISCUSSION TITLE
Add a scaled vector API naming poll

### DISCUSSION THREAD

**Discussion by LeXXik** [playcanvas/engine#5707] [Polls]
A poll for public API name in #5702.

**Comment by kungfooman** [comment]
Maybe just my old C engine sentimentalism:

https://github.com/ioquake/ioq3/blob/b1e6ef14254fdcb77cbe2f602c22549d3fb0eb78/code/qcommon/q_math.c#L826-L830

`mulAdd` - first we multiply the vector, then adding result to `this`
